src/index.html

```diff
@@ -7,8 +7,80 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo"
+         href="#home"
+      >
+        <img src="images/logo.png"
+             alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a class="nav-link is-active"
+              href="#apple"
+            >
```

Comment:
Fix it everywhere please
```suggestion
            <a
              class="nav-link is-active"
              href="#apple"
            >
```
OR

If there are less than 3 attributes

```suggestion
            <a  class="nav-link is-active" href="#apple">
```